qa-tests-backend/src/test/groovy/ImageSignatureVerificationTest.groovy

```diff
@@ -143,6 +145,15 @@ QC+pUMTUP/ZmrvmKaA+pi55F+w3LqVJ17zwXKjaOEiEpn/+lntl/ieweeQ==
             SAME_DIGEST_WITH_SIGNATURE,
     ]
 
+    static final private List<String> IMAGE_DIGESTS = [
+            "sha256:bc217643f9c04fc8131878d6440dd88cf4444385d45bb25995c8051c29687766",
+            "sha256:d12d420438235ccee3f4fcb72cf9e5b2b79f60f713595fe1ada254d10167dfc6",
+            "sha256:743cf31b5c29c227aa1371eddd9f9313b2a0487f39ccfc03ec5c89a692c4a0c7",
+            "sha256:b73f527d86e3461fd652f62cf47e7b375196063bbbd503e853af5be16597cb2e",
+            "sha256:dd2d0ac3fff2f007d99e033b64854be0941e19a2ad51f174d9240dda20d9f534",
+            "sha256:dd2d0ac3fff2f007d99e033b64854be0941e19a2ad51f174d9240dda20d9f534",
```

Comment:
Maybe you can give a hint what images are under this shas

Reply:
I've extracted those to their own variables to make things more clear.